main.tex

```diff
@@ -1030,7 +1045,7 @@
 \item  the person conducting the trade, profession or business is a fulltime resident of the Lot and only
 \newpage
 
-operates a home office with a maximum of one employee; and
+operates a home office with a maximum of two co-located human employees; and
```

Comment:
A friend of mine has epilepsy and always has 2 staff accompanying her, to manage her seizures. Limiting people to one employee is needlessly restrictive.